platform/Task.roc

```diff
@@ -208,3 +210,36 @@ batch = \current -> \next ->
         f <- next |> await
 
         map current f
+
+## Apply a task repeatedly to a list of items, and return a list of the resulting values
+##
+## ```
+## authors : List ID
+## getAuthor : ID -> Task Author [DbError]
+##
+## getAuthors : Task (List Author) [DbError]
+## getAuthors = Task.list authors getAuthor
+## ```
+##
+list : List a, (a -> Task c d) -> Task (List c) d
```

Comment:
What if we call this `Task.seq` (for "sequential") and have it just be `List (Task ok err) -> Task ok err`?

That way you can transform the list however you want (e.g. with `List.map`) first, and then `|> Task.seq` to run them all sequentially.

(We might also want a `Task.concurrent` or something in the future as well, once we can support that in the host.)

Reply:
What do we do then with the intermediate values between tasks? Should we throw them away and keep the final value? Or maybe restrict it to `Task {} err`'s so there is no value returned?

Reply:
Oops, I guess that should be:

```elm
List (Task ok err) -> Task (List ok) err
```

Reply:
Updated to `Task.seq` ✅